pom.xml

```diff
@@ -13,11 +13,25 @@
 
     <modules>
         <module>symphony-bdk-legacy</module>
```

Comment:
Should we create another profile for legacy? it might be confusing if legacy is included in 2.0

Reply:
Profile `2.0` is by default not activated since we don't want to start releasing related modules, but we still want to release legacy modules in any case. So yes, for me legacy modules are also part of the 2.0 (like JUnit5/Vintage : https://junit.org/junit5/docs/current/user-guide/#migrating-from-junit4)

Reply:
Well, finally I introduced a `legacy` profile as well 😅 However it is activated by default which is not the case for the `2.0` one